src/templates/blog/post.js

```diff
@@ -22,14 +22,22 @@ export const query = graphql`
         date
         retina_cover
         title
+        description
       }
     }
   }
 `
 
-const BlogPostTemplate = ({ author, date, html, retinaCover, title }) => (
+const BlogPostTemplate = ({
+  author,
+  date,
+  html,
+  retinaCover,
+  title,
+  description,
```

Comment:
Do you mind keeping the alphabetic order? :grimacing: 

```suggestion
  author,
  date,
  description,
  html,
  retinaCover,
  title,
```